app/controllers/admin/truework_income_verifications_controller.rb

```diff
@@ -17,26 +17,31 @@ def new
 
   # POST /applicants/[ID]/income_verifications/create
   def create
-  	v = TrueWorkIncomeValidation.validate_applicant(@applicant, params['true_work_income_validation'])
+  	v = TrueworkIncomeVerification.verify_applicant(@applicant, params['truework_income_verification'])
   	redirect_to admin_applicant_truework_income_verification_path(@applicant, v)
   end
 
-  # GET /applicants/:applicant_id/income_verifications/:id
+  # GET /truework_income_verifications/:id
   def show
   end
 
   # GET /applicants/:applicant_id/income_verifications
   def index
-  	@verifications = Validation.where(applicant: @applicant)
+  	@verifications = Verification.where(applicant: @applicant)

```

Comment:
I didn't change the logic here, but on review, it looks like it might be wrong.  The `TrueworkIncomeVerificationsController::index` method should display all `TrueworkIncomeVerification`s, not all `Verification`s (which might include other types), right?